src/OMSimulatorPython/component.py

```diff
@@ -1,15 +1,16 @@
 from lxml import etree as ET
 from OMSimulator.cref import CRef
 from OMSimulator.values import Values
-
+from OMSimulator.variable import Variable
 from OMSimulator import namespace
 
 
 class Component:
-  def __init__(self, name: CRef, fmuPath: str, connectors = list()):
+  def __init__(self, name: CRef, fmuPath: str, connectors = list(), unitDefintions = list()):
```

Comment:
Using a mutable default argument such as a list can lead to unexpected behavior. If the list is modified in one instance of the class, the changes will persist across all instances of the class that use the default value. This is because the default value is evaluated only once at the time the function is defined, not each time the function is called.